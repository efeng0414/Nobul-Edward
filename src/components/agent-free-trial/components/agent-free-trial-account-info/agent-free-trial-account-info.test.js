import React from "react";
import renderer from "react-test-renderer";
import { IntlProvider } from "react-intl";
import AgentFreeTrialAccountInfoWrapped from "./index";

describe("AgentFreeTrialAccountInfo Component", () => {
  const {
    WrappedComponent: AgentFreeTrialAccountInfo
  } = AgentFreeTrialAccountInfoWrapped;

  const componentProps = {
    intl: global.getIntl(),
    firstName: "Agent",
    trialCompleted: true
  };

  const componentJSX = <AgentFreeTrialAccountInfo {...componentProps} />;

  describe("Component Snapshot", () => {
    const snapshotJSX = (
      <IntlProvider {...global.intl}>{componentJSX}</IntlProvider>
    );

    const componentSnapshot = renderer.create(snapshotJSX).toJSON();
    it("Should match stored snapshot", () => {
      expect(componentSnapshot).toMatchSnapshot();
    });
  });
});
