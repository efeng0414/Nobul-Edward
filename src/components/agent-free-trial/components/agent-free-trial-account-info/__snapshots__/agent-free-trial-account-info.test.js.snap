// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AgentFreeTrialAccountInfo Component Component Snapshot Should match stored snapshot 1`] = `
<div
  className="free-trial-account-info"
>
  <div
    className="free-trial-account-info__hello"
  >
    Hello Agent,
  </div>
  <div
    className="free-trial-account-info__membership"
  >
    Membership:
  </div>
  <div
    className="free-trial-account-info__membership__type"
  >
    nobul Premium
    <span
      className="free-trial-account-info__membership__freeTrial"
    />
  </div>
</div>
`;
